blocks/author-bio-block/features/author-bio/default.jsx

```diff
@@ -98,167 +98,189 @@ const AuthorBio = () => {
         ? author.social_links.reduce((result, socialLink) => {
           if (socialLink.site && socialLink.url && socialLink.url.length > 0) {
             let socialButton;
+            let linkTitle;
             const constructedURL = constructSocialURL(socialLink.site, socialLink.url);
 
+            const MediaLink = ({ children, label, ...otherProps }) => (
+              <MediaLinksStyled
+                href={constructedURL}
+                target="_blank"
+                rel="noreferrer noopener"
+                aria-label={`Click ${label} link`}
```

Comment:
You shouldn't use the word "link" within the description as this is handled by a screen reader and causes duplication, the same goes for "click" uses are told they are on a link and this also implies you need to "click" it, which isn't always the case for non mouse users


This is what a screen reader will read out when they see this link now - "link, image, Click Twitter link"
<img width="688" alt="PEN-676 VoiceOver link value" src="https://user-images.githubusercontent.com/868127/107504995-f766b500-6b93-11eb-9d83-cde1e4285627.png">

Don't worry too much about the image part, that is going to be taken care of in the SVG update ticket.

The idea here is we want to provide meaningful text to someone who is unable to gain the same visual experience we have from looking at the page.
Something like - [Vinnie Olsen] on [Twitter] - and replacing the name with the author and twitter with the correct value.

Reply:
@matthewroach updated the label, refactored a little and added more unit test coverage 👍  Let me know if that looks better!